docs/guides-publish.md

```diff
@@ -35,104 +37,98 @@ This app has 3 environments defined:
 
 - An ENS registry: `0x98df287b6c145399aaa709692c8d308357bc085d`
-- An app name (repository and registry of aragonPM): `app.open.aragonpm.eth`
-- An Ethereum websockets provider: `wss://rinkeby.eth.aragon.network/ws` - to **read** from the blockchain
+- An app name (repository and registry of aragonPM): `app.aragonpm.eth`
 - An Ethereum network: `rinkeby`
 
-The `rinkeby` network is further defined in `truffle.js`, and has:
-
-- An Ethereum provider address: `https://rinkeby.infura.io` (to **write** to the blockchain)
-- An Ethereum account: `0xb41...6eE7` (which is the first account generated from the `DEFAULT_MNEMONIC` variable, to learn how to use a different account follow the [troubleshooting guide](/docs/guides-faq.html#set-a-private-key))
-
 > **Note**<br>
-> The `default` environment which points to `localhost` does not have an ENS Registry address specified because aragonCLI will default the value to `0xB9462EF3441346dBc6E49236Edbb0dF207db09B7` (the ENS Registry pre-deployed on the local development chain).
+> The `default` environment which points to `localhost` does not have an ENS Registry address specified because the Buidler plugin will default the value to `0xB9462EF3441346dBc6E49236Edbb0dF207db09B7` (the ENS Registry pre-deployed on the local development chain).
 
 ## Publish a major version: content + contract
 
-To publish on aragonPM we will use [`aragon apm publish`](/docs/cli-apm-commands#aragon-apm-publish) command.
-
-Command:
+To publish on aragonPM we will use the following buidler task:
 
 ```sh
-npx aragon apm publish major --environment rinkeby
+npx buidler publish major --network rinkeby
 ```
 
 This will:
 
-<span>![*](/docs/assets/check.svg) Apply version bump (major).</span>
-
-<span>![*](/docs/assets/check.svg) _Compile_ and _deploy_ the app's contract (by default the output lives in `build`)</span>
-
-<span>![*](/docs/assets/check.svg) _Build_ the app's [frontend (by default the output lives in `dist`)](#building-frontends)</span>
-
-<span>![*](/docs/assets/check.svg) Generate application artifact.</span>
-
-<span>![*](/docs/assets/check.svg) Publish the app to the **rinkeby** environment.</span>
+- Apply version bump (major).
+- _Compile_ and _deploy_ the app's contract (by default the output lives in `build`)
+- _Build_ the app's [frontend (by default the output lives in `dist`)](#building-frontends)
+- Generate application artifact.
+- Publish the app to the **rinkeby** environment.
+- Upload the app's frontend to the IPFS server.
 
 Sample output:
 
 ```sh
- > npx aragon apm publish major --environment rinkeby
-
- ✔ Successfully published app.open.aragonpm.eth v1.0.0:
- ℹ Contract address: 0xE636bcA5B95e94F749F63E322a04DB59362299F1
- ℹ Content (ipfs): QmR695Wu5KrHNec7pRP3kPvwYihABDAyVYdX5D5vwLgxCn
- ℹ Transaction hash: 0x3d752db29cc106e9ff98b260a90615921eb32471425a29ead8cbb830fb224d8
+main     |   App name:          app.aragonpm.eth
+main     |   Initial version:   1.0.0
```

Comment:
cc @0xGabi do we always print `Initial version` or will it turn to just `Version` on later publishes?

Reply:
I just tested it. It displays “Publish new version” the following times.